src/main/java/com/carsharing/model/Car.java

```diff
@@ -0,0 +1,24 @@
+package com.carsharing.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "cars")
+public class Car {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String model;
+    private String brand;
+    private CarType type;
+    private int inventory;
+    private BigDecimal dailyFee;
```

Comment:
maybe add @Column(name = "daily_fee")?

Reply:
why only it? Maybe then for every?

Reply:
because dailyFee is only variable that named more than 1 word